Core/src/main/java/com/plotsquared/core/util/StringMan.java

```diff
@@ -320,4 +323,14 @@ public static boolean contains(String name, char c) {
         return col;
     }
 
+    public static final Pattern ALLOWED_PATTERN = Pattern.compile("(?<quoted>\"[\\w ]+\")|(?<single>\\w+)");
+
+    public static List<String> splitMessage(String message){
```

Comment:
This method should be documented. It's unclear how `message` is split.

Reply:
is documented with a table of examples.